### DISCUSSION TITLE
copy entire folder structure to root?

### DISCUSSION THREAD

**Discussion by teknoel** [nguyenbatranvan/vite-multiple-assets#33] [General]
Hi there, I'm wondering if your plugin can copy an entire folder structure, which has nested folders and files inside, to the output path but maintain the entire folder hierarchy intact? Just like the actual "public" folder currently works with vite.

From what I can tell, your plugin would copy all the files to the input directory to the output path, effectively flattening the hierarchy of the original source structure

`{ input: "content/**", output: "/content" }`

**Comment by nguyenbatranvan** [comment]
I think I will add an option to flatten for each output. Maybe like this:

```
{ input: "content/**", output: "/a/b", flattern: false}
```

To flatten or copy the entire folder over. 

For your simple case, if you want to get the parent folder, just do the following:

```
DynamicPublicDirectory(["{\x01, content}/**"], { ssr : false })
```

**Comment by nguyenbatranvan** [comment]
u can see from #34

**Reply by teknoel** [reply]
This looks like a good option.

**Reply by nguyenbatranvan** [reply]
The new version will be released in the next few days. Thank you for using the plugin.

**Reply by nguyenbatranvan** [reply]
version `2.2.1` released
